recipes/blpapi/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "blpapi" %}
+{% set version = "3.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://bloomberg.bintray.com/pip/simple/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 4c3cbf3fb63fd338360d55188f6a59568023be6e16531d4bd307ac59a297fd96
+  - url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-linux.tar.gz
+    sha256: c6c425f714b461ec618e6900bda8d9b4fc919892538ccfbf8c9a774e9e234fee
+    folder: blpapi_cpp_linux
+  - url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-windows.zip
+    sha256: d62e3f109665f027e2ba3117d015c06a20d5e79e273777ef87915c82311d3fec
+    folder: blpapi_cpp_windows
```

Comment:
if you really need to download files according to the platform, please use selectors like:
```
url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-linux.tar.gz  # [linux]
sha256: c6c425f714b461ec618e6900bda8d9b4fc919892538ccfbf8c9a774e9e234fee  # [linux]
url: https://software.tech.bloomberg/BLPAPI-Stable-Generic/blpapi_cpp_3.8.18.1-windows.zip  # [win]
sha256: d62e3f109665f027e2ba3117d015c06a20d5e79e273777ef87915c82311d3fec  # [win]
```